scripts/install_v8.sh

```diff
@@ -0,0 +1,6 @@
+WD=`pwd`
+sudo apt install nodejs && \
```

Comment:
Using ``apt-get`` is more conventional way to install something. In latest distributions they come together, but in older ones that might be an issue.